source/initial-dns-seedlist-discovery/tests/README.rst

```diff
@@ -25,19 +25,15 @@ these tests::
   _mongodb._tcp.test6.test.build.10gen.cc.  86400  IN SRV  27017  localhost.build.10gen.cc.
   _mongodb._tcp.test7.test.build.10gen.cc.  86400  IN SRV  27017  localhost.build.10gen.cc.
   _mongodb._tcp.test8.test.build.10gen.cc.  86400  IN SRV  27017  localhost.build.10gen.cc.
-  _mongodb._tcp.test9.test.build.10gen.cc.  86400  IN SRV  27017  localhost.build.10gen.cc.
```

Comment:
Could you add a note to initial-dns-seedlist-discovery/tests/README.rst, where it says that test4 is deliberately omitted? Also say that test9 is omitted because it was deleted during development of the tests. That'll help avoid confusion.